app/models/policy_event.rb

```diff
@@ -28,9 +28,9 @@ def self.create_events(target, event, result)
       # TODO: username,
       )
 
-      pe.host_id = target.respond_to?(:host) && !target.host.nil? ? target.host.id : nil
-      pe.ems_id = target.respond_to?(:ext_management_system) && !target.ext_management_system.nil? ? target.ext_management_system.id : nil
-      pe.ems_cluster_id = target.respond_to?(:owning_cluster) && !target.owning_cluster.nil? ? target.owning_cluster.id : nil
+      pe.host_id = target.try(:host)&.id
+      pe.ems_id = target.try(:ext_management_system)&.id
+      pe.ems_cluster_id = target.try(:owning_cluster)&.id
```

Comment:
Okay... so the only reason for this "drive by comment" is I say this PR come in, and I mostly assumed I was going to be seeing a bunch the `&.` operator... which I have ranted about disliking particularly (and I won't do that here).

However, I was pleasantly surprised to see that you mostly used `.try` instead...

... except for these three lines... where you use BOTH!?

You aren't even consistent with yourself on the same line...

Reply:
:trollface:

Reply:
yea, sorry for the wondefully used `&.` :trollface: 

It was saying use an `id` if this object responded to the `host`
I guess I could have used `.try(:host).try!(:id)` for these, but then rubocop would have complained and asked me to use `&.`... sigh

To be honest, my issue with these was that I wanted to see `target.try(:host_id)` instead... but I didn't want to research it enough to nail down whether these were `belongs_to` or `has_one`